### ISSUE TITLE
Narrator screen reader is not announcing anything when it encounters the Upload button

### ISSUE THREAD

**Issue by guybark** [Microsoft/BotFramework-WebChat#1531]
Turn on the Narrator screen reader, and tab to the Upload button. Narrator says nothing, and so the customer can't know where they are. The attached image shows the Keros for Desktop tool reporting a failure with that UI. Two issues are (i) the element isn't actually a Button, rather it's a Text element, and (ii) it has an empty Name property. 

![webchatbutton](https://user-images.githubusercontent.com/14315611/50357842-2df09a00-050c-11e9-82ed-3b3ae4a0a5b6.png)

**Comment by corinagum** [comment]
@guybark Thanks for filing these issues! This fix should be in the next iteration of Web Chat, per this bug: https://github.com/Microsoft/BotFramework-WebChat/issues/1308. The next release is (hopefully) imminent. Please let us know if your v4 bot still has this problem in the next update.

**Comment by corinagum** [comment]
4.3 is out and this should be resolved. Please feel free to open a new issue or comment here if there are ongoing problems. :)
